backend/models/customer.js

```diff
@@ -3,15 +3,55 @@ const ClientBase = require("./client");
 
 const Schema = mongoose.Schema;
 
+var address = {
+  addressName: { type: String },
+  name: { type: String },
+  addressLine: { type: String },
+  city: { type: String },
+  state: { type: String },
+  zipCode: { type: String },
+  phone: { type: String },
+};
+
+var shoppingList = {
+  id: { type: String },
+  title: { type: String },
+  wishedProducts: [],
+};
+
+var order = {
```

Comment:
Order should be created as a seperate model. It interacts with multiple models.

Reply:
Orders now have their own seperate model.